libafl/src/events/broker_hooks/mod.rs

```diff
@@ -157,7 +157,7 @@ where
                 client_stats_manager.update_client_stats_for(client_id, |client_stat| {
                     client_stat.update_user_stats(name.clone(), value.clone());
                 });
-                client_stats_manager.aggregate(name);
+                client_stats_manager.aggregate(name.clone());
```

Comment:
Can't we pass in a reference to `name` instead of cloning?

Reply:
Ok. I'll check it.